prince-of-versions/src/main/java/co/infinum/princeofversions/JsonConfigurationParser.java

```diff
@@ -66,7 +66,12 @@
     /**
      * Android key
      */
-    private static final String ANDROID = "android2";
+    private static final String ANDROID = "android";
+
+    /**
+     * Android2 key
+     */
+    private static final String ANDROID2 = "android2";
```

Comment:
Let's call them `ANDROID2` => `ANDROID_KEY` and `ANDROID` => `ANDROID_FALLBACK_KEY`